backend/Epicenter.Persistence/Repositories/PersonRepository.cs

```diff
@@ -10,31 +9,34 @@ namespace Epicenter.Persistence.Repositories
 {
     public class PersonRepository : IPersonRepository
     {
-        private readonly Mapper<Person> _mapper;
+        private readonly EpicenterContext _context;
 
-        public PersonRepository(Mapper<Person> mapper)
+        public PersonRepository(EpicenterContext context)
         {
-            _mapper = mapper;
+            _context = context;
```

Comment:
get a better name, at least something like _dbContext, because just "_context" is too abstract